apm-sniffer/apm-sdk-plugin/mysql-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v5/define/CallableInstrumentation.java

```diff
@@ -67,6 +66,11 @@
     }
 
     @Override protected ClassMatch enhanceClass() {
-        return byMultiClassMatch(ENHANCE_CLASS, "com.mysql.jdbc.cj.CallableStatement");
+        return byName(ENHANCE_CLASS);
+    }
+
+    @Override
+    protected String[] witnessClasses() {
```

Comment:
I advise that this method should remove. because this method had implemented in its parent class

Reply:
i am sorry,i've forgotten